codemods/babel-plugin-codemod-object-assign-to-object-spread/.npmignore

```diff
@@ -1,3 +1,4 @@
 src
 test
-*.log
```

Comment:
I suggest we keep the `*.log` here. In circumstances where we have to manually publish npm packages and the first publish attempt is failed, npm will generate a debug log `npm-debug.log`, if we forget to remove that during the second publish attempt, this log will be unfortunately published.